Buildings/Fluid/Boilers/Data/Lochinvar/Crest/FBdash4001.mo

```diff
@@ -0,0 +1,17 @@
+within Buildings.Fluid.Boilers.Data.Lochinvar.Crest;
+record FBdash4001 "Specifications for Lochinvar Crest FB-4001 boiler"
+  extends Buildings.Fluid.Boilers.Data.Lochinvar.Crest.Curves(
+    Q_flow_nominal = 1126272.122,
+    VWat = 0.760867769,
+    mDry = 1725.918968,
+    dT_nominal = 12.222222,
```

Comment:
This is the only record with 12.222. Is this correct or should it also be 11.1111?